packages/oauth/end_of_redirect_response.js

```diff
@@ -12,6 +12,15 @@
     }
   }
 
+  if (config.error) {
+    try {
+      localStorage[config.storagePrefix + "error"] = config.error;
+      localStorage[config.storagePrefix + "error_description"] = config.error_description;
+    } catch (err) {
+      console.log("Error storing OAuth error in localStorage", err);
```

Comment:
Please do not use console.log since it's a thread blocker, use Log.warn/error instead